.github/workflows/dispatch.yml

```
@@ -0,0 +1,19 @@
+---
+name: deploy
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.CON_PCA_ACCESS_TOKEN }}
+          repository: cisagov/con-pca-cicd
+          event-type: deploy-sandbox
+          client-payload: '{"trigger": "con-pca-tasks-docker"}'
```

We prefer to alphabetize whenever possible:
```suggestion
          client-payload: '{"trigger": "con-pca-tasks-docker"}'
          event-type: deploy-sandbox
          repository: cisagov/con-pca-cicd
          token: ${{ secrets.CON_PCA_ACCESS_TOKEN }}
```

---

@dav3r resolved with [b9d2545](https://github.com/cisagov/con-pca-tasks-docker/pull/4/commits/b9d254556aa6bef4f89b22ad7a90008a55e0de6b)